ocs_ci/deployment/deployment.py

```diff
@@ -1048,40 +1049,94 @@ def setup_local_storage(storageclass):
             raise NotImplementedError(
                 "LSO Deployment for VMDirectPath is not implemented"
             )
+    if (ocp_version >= "4.6") and (ocs_version >= "4.6"):
```

Comment:
So in case one of OCP or OCS is less than 4.6 we will go with else block right? So this approach is applicable only when both OCP and OCS are >= 4.6 ?

Reply:
Yes with lower version then 4.6 we will go with localvolume creation yes the appoach is applicable for 4.6 version only